src/main/scala/gui/EventsScene.scala

```diff
@@ -0,0 +1,105 @@
+  textField.prefWidth = textFieldWidth
+
+  val searchButton = new EventsButton("Search", offX + textFieldWidth, offYSearch)
+
+  val resetButton = new EventsButton("Show all", 2 * offX + textFieldWidth + buttonWidth, offYSearch)
+
+  val list = new EventsList(scraper.getEventList, offX, offYList)
+
+  val observeButton = new EventsButton("Monitor concert", offX, offYDown)
+
+  val getLinkButton = new EventsButton("Go to Web Page", 2 * offX + buttonWidth, offYDown)
+
+  val monitorLabel = new Label("Monitored events:")
+  monitorLabel.layoutX = 2 * offX + listWidth
+  monitorLabel.layoutY = offYSearch
+
+  val monitorList = new EventsList(List[Event](), 2 * offX + listWidth, offYList)
+
+  val stopMonitorButton = new EventsButton("Stop monitoring", 2 * offX + listWidth, offYDown)
+
+  content = List(monitorLabel, monitorList, label, textField, searchButton, list,
+    observeButton, getLinkButton, stopMonitorButton, resetButton)
+
+  searchButton.onAction = handle {
+    list.items = ObservableBuffer(SearchEngine.searchForEvents(scraper.getEventList, textField.getText()))
+  }
+
+  resetButton.onAction = handle {
+    list.items = ObservableBuffer(scraper.getEventList)
+  }
+
+  observeButton.onAction = handle {
+    val selectedEvent = list.selectionModel.apply.getSelectedItem
+    if (selectedEvent == null) {
+      new EventsAlert(AlertType.Error, "You must select event to monitor!", "Not selected", stage).showAndWait()
+    } else if (selectedEvent.ticketsAvailable()) {
+      new EventsAlert(AlertType.Error, "Cannot monitor event which has avalaible tickets!", "Cannot monitor", stage).showAndWait()
+    } else if (!monitor.startMonitoring(selectedEvent)) {
+      new EventsAlert(AlertType.Error, "This event is already being monitored!", "Already monitoring!", stage).showAndWait()
+    } else {
+      println("Monitoring  event: " + selectedEvent.name)
+      monitorList.items.getValue.add(selectedEvent)
+    }
+  }
+
+  getLinkButton.onAction = handle {
+    val selectedEvent = list.selectionModel.apply.getSelectedItem
+    if (selectedEvent == null) {
+      new EventsAlert(AlertType.Error, "You must select event to get its link from events list!", "Not selected", stage).showAndWait()
+    } else {
+      println("Event's URL: " + selectedEvent.getUrl)
+      if (Desktop.isDesktopSupported && Desktop.getDesktop.isSupported(Desktop.Action.BROWSE))
+        Desktop.getDesktop.browse(new URI(selectedEvent.getUrl))
+    }
+
+
+  }
+  stopMonitorButton.onAction = handle {
+    val selectedEvent = monitorList.selectionModel.apply.getSelectedItem
+    if (selectedEvent == null) {
+      println("Nothing selected!")
+      new EventsAlert(AlertType.Error, "You must select event to stop from monitored events list!", "Not selected!", stage).showAndWait()
+    } else {
+      monitor.stopMonitoring(selectedEvent)
+      println("Stopping monitoring event:" + selectedEvent.name)
+      monitorList.items.getValue.remove(selectedEvent)
```

Comment:
This could be a method in EventsList, would have better encapsulation
By the way, i think monitoredList would tell that it is a list of events that are monitored, rather than list of some monitor object